packages/api/src/axelar-query/types.ts

```diff
@@ -20,3 +28,83 @@ export type EstimateGasFeeResponse = {
   apiResponse: string;
   isExpressSupported: boolean;
 };
+
+export type Environment = "devnet" | "testnet" | "mainnet";
```

Comment:
use [this type](https://github.com/axelarnetwork/axelarjs/blob/76f4a83a5c808753e7e0567c9862c5ca68b3a656/packages/cosmos/src/types.ts#L3) instead.